README.md

```diff
@@ -1,9 +1,111 @@
+* Confirm the feature is running in the Admin Console or the Karaf command line interface.
+
+## Configuring Replication Nodes
+This version of Replication supports nodes created for DDF based applications as well as webHDFS based systems.
+
+#### Configuring DDF Nodes
+To configure replication to/from a DDF node, a node must first be created. This can be done in a few steps:
+* Navigate to https://<host>:<port>/admin/replication/#/nodes and click the + 
+* On the Create new Node screen, fill out the following fields:
+  * Name: The display name to give the node. 
+  * Hostname: The hostname of the node.
+  * Port: The port where the service can be found. This is `8993` by default in DDF.
+  * Root Context: The path under which replication services can be found. This is `services` by default in DDF.
+  
+#### Configuring WebHDFS Nodes
+To configure replication to/from a HDFS node, a node must first be created. This can be done in a few steps:
+* Navigate to https://<host>:<port>/admin/replication/#/nodes and click the + 
+* On the Create new Node screen, fill out the following fields:
+  * Name: The display name to give the node. 
+  * Hostname: The hostname of the node.
+  * Port: The port where the service can be found. This is `9870` by default for HDFS Namenodes.
+  * Root Context: This is the full path after the port for any webHDFS URL.
+
+* localhost webHDFS example configuration : 
+```
+To write to directory /user/tonystark on a local webHDFS instance
+- The webHDFS URL for that directory might look like: http://localhost:9870/webhdfs/v1/user/tonystark
+- The hostname would be: localhost
+- The port would be: 9870
+- The root context would be /webhdfs/v1/user/tonystark/
+```  
+* remote webHDFS example configuration:  
+```
+To write to directory /user/tonystark on a remote AWS HDFS installation
+- The webHDFS URL for that directory might look like: https://testnode1.us-gov-west-1.compute.amazonaws.com:8443/gateway/default/webhdfs/v1/user/tonystark
+- The hostname would be: testnode1.us-gov-west-1.compute.amazonaws.com
+- The port would be: 8443
+- The root context would be /gateway/default/webhdfs/v1/user/tonystark/
+```  
+
+## Configuring Replication
+After the nodes are created, a replication can be configured. This can be done in a few steps:
+* Navigate to https://<host>:<port>/admin/replication click the Add Replication 
+* On the New Replication Setup screen, fill out the following fields:
+  * Replication Name: The display name for this replication.
+  * Source Node: The node in the dropdown to use as the source (replicating FROM this source).
+  * Destination Node: The node in the dropdown to use as the destination (replicating TO this node).
+  * Bidirectional: Select this checkbox if replication should synchronize the contents of source and destination nodes.
+  * Filter: A previously used Intrigue query can be selected from the dropdown, otherwise enter CQL into the Filter field.
```

Comment:
Doesn't the Intrigue query need to be saved to be selected? That was what I thought I experienced. If so, I'd mention that the user would need to first go into Intrigue and save their search before it is available in the Filter.

Reply:
I'm not sure. What do you mean by saved? For instance, I usually go to the workspaces menu and just enter `*` into the box and then in replication setup I can click `*` query and it populates `anytext ILIKE '%'` for the CQL box. Should I put this as an example maybe?

Reply:
You can save workspaces in Intrigue by clicking the 💾 icon next to the name in the top left corner. I don't think the workspace has to be saved for the search term to appear in replication though, as long as the search has already been run.